src/redux/reducers/userReducer.ts

```diff
@@ -0,0 +1,18 @@
+import initialState from '../initialState';
+import { Action, UserActions } from '../actions/userActions';
+
+const userReducer = (
+  state = initialState,
+  action: Action<UserActions, string>
```

Comment:
For future reference: if you'd like to add multiple users the type here would be:
```typescript
Action<UserActions, string | string[]>
```